src/libnnstreamer-edge/nnstreamer-edge-internal.c

```diff
@@ -929,8 +915,9 @@ _nns_edge_accept_socket_async_cb (GObject * source, GAsyncResult * result,
     _nns_edge_close_connection (conn);
   }
 
-  g_socket_listener_accept_socket_async (socket_listener, eh->cancellable,
-      (GAsyncReadyCallback) _nns_edge_accept_socket_async_cb, eh);
+  if (eh->listener)
+    g_socket_listener_accept_socket_async (eh->listener, NULL,
+        (GAsyncReadyCallback) _nns_edge_accept_socket_async_cb, eh);
```

Comment:
Q: The `listener` pointer in handle object and `socket_listener` in `_nns_edge_accept_socket_async_cb()` callback function should be the same object. If then, is checking logic needed in `_nns_edge_accept_socket_async_cb()` function?

Reply:
When releasing edge handle, it closes listener. This will make socket_listener as closed.
This is async callback and when accessing the expired edge handle, it will be crashed.

Reply:
You're right. I will wait for your new patch to fix it. Thank you. :+1: